### ISSUE TITLE
Front page links

### ISSUE THREAD

**Issue by MSchirp** [jetlej/HE-6.0-Bugs#109]
I think the more topics link here should lead to: http://hiexistence.com/discussions/, not http://hiexistence.com/topics.

![screen shot 2015-10-21 at 13 17 21](https://cloud.githubusercontent.com/assets/13124056/10635037/1a266690-77f6-11e5-8b37-31ac05481a4f.png)

The more quotes link,

![screen shot 2015-10-21 at 13 17 45](https://cloud.githubusercontent.com/assets/13124056/10635050/2cc0b2f6-77f6-11e5-9bba-dda997f98a9e.png)

Leads to the quotes page, 

![screen shot 2015-10-21 at 13 19 17](https://cloud.githubusercontent.com/assets/13124056/10635072/5df5f35e-77f6-11e5-9bc4-2e461d2757ad.png)

Basically a dead end. We don't really want people to go there.

I think either removing this link altogether and just showing a random quote there every time the page is refreshed, or replace the link with a button that replaces the quote with a random new one.

**Comment by jetlej** [comment]
Fixed
